scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1309,3 +1312,54 @@ <h6 class="card-subtitle mb-2 text-muted"><span class="badge badge-warning">Part
   </tbody>
 </table>
 {% endmacro %}
+
+{% macro somatic_variant_table(variants) %}
+  <div class="card" style="border-width: 5px; display: block;">
+    <div class="card-header">Summary table</div>
+    <div class="card-body">
+      <div class="row d-flex align-items-center">
+       <table id="panel-table" class="table table-sm table-bordered" style="background-color: transparent">
+        <thead>
+          <tr class="table-secondary">
+            <th>Gene Symbol</th>
+            <th>Transcript</th>
+            <th>HGVS c.</th>
+            <th>HGVS p.</th>
+            <th>Tumor VAF%</th>
+            <th>ACMG</th>
+          </tr>
+        </thead>
+        <tbody>
+        {% for acmg in ["P","LP", "VUS", "LB", "B", ] %}
+          {% for variant in variants|selectattr("acmg_classification.short", "equalto", acmg)|sort(attribute='first_rep_gene.hgnc_symbol') %}
+            {{ variant_table_line(variant) }}
+          {% endfor %}
+        {% endfor %}
+        {% for variant in variants|selectattr("acmg_classification.short", "undefined")|sort(attribute='first_rep_gene.hgnc_symbol') %}
+            {{ variant_table_line(variant) }}
+          {% endfor %}
+        </tbody>
+       </table>
+      </div>
+    </div>
+  </div>
+{% endmacro %}
+
+{% macro variant_table_line(variant) %}
+<tr>
+  <td>{{ variant.first_rep_gene.hgnc_symbol or variant.first_rep_gene.hgnc_id }}</td>
```

Comment:
I've tried it locally with the cancer demo sample and one SNV and SV and got this:

Traceback (most recent call last):
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1822, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask_cors/extension.py", line 165, in wrapped_function
    return cors_after_request(app.make_response(f(*args, **kwargs)))
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1820, in full_dispatch_request
    rv = self.dispatch_request()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/app.py", line 1796, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/utils.py", line 90, in decorated_function
    return render_template(template_name, **context)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/templating.py", line 147, in render_template
    return _render(app, template, context)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/flask/templating.py", line 130, in _render
    rv = template.render(context)
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/jinja2/environment.py", line 1301, in render
    self.environment.handle_exception()
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/cases/templates/cases/case_report.html", line 1359, in top-level template code
    <span class="badge badge-pill badge-{{variant.acmg_classification['color'] if variant.acmg_classification['color'] else 'secondary'}}" title="{{variant.acmg_classification['code']}}">{{variant.acmg_classification['short'] }}</span>
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/templates/report_base.html", line 20, in top-level template code
    {% block body %}
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/cases/templates/cases/case_report.html", line 61, in block 'body'
    {{ pinned_panel() }}
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/cases/templates/cases/case_report.html", line 345, in template
    {{ somatic_variant_table(variants.suspects_detailed) }}
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/jinja2/runtime.py", line 777, in _invoke
    rv = self._func(*arguments)
  File "/Users/chiararasi/Documents/work/GITs/scout/scout/server/blueprints/cases/templates/cases/case_report.html", line 1338, in template
    {% for variant in variants|selectattr("acmg_classification.short", "undefined")|sort(attribute='first_rep_gene.hgnc_symbol') %}
  File "/Users/chiararasi/miniconda3/envs/py38/lib/python3.8/site-packages/jinja2/filters.py", line 409, in do_sort
    return sorted(value, key=key_func, reverse=reverse)
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'hgnc_symbol'

I'm sure it's an easy fix!